test/extended/testdata/statusfail-fetchsource.yaml

```diff
@@ -1,14 +1,14 @@
 kind: BuildConfig
 apiVersion: v1
 metadata:
-  name: failstatus-fetchsource
+  name: statusfail-fetchdockersource
 spec:
   source:
     git:
-      uri: "https://failure/s2i/status"
+      uri: "https://failure/docker/status"
   strategy:
+    type: Docker
     dockerStrategy:
       from:
         kind: DockerImage
-        name: ruby:latest
-    type: Docker
+        name: centos/ruby-23-centos7:latest
```

Comment:
since you named the other file statusfail-fetchsources2i.yaml, name this one statusfail-fetchsourcedocker.yaml